src/sentry/tasks/sentry_apps.py

```diff
@@ -96,13 +96,24 @@ def send_alert_event(event, rule, sentry_app_id):
         data=data,
     )
 
-    safe_urlopen(
+    response = safe_urlopen(
         url=sentry_app.webhook_url,
         data=request_data.body,
         headers=request_data.headers,
         timeout=5,
```

Comment:
Does this throw in case of timeout? If yes you would not record the metric there. Is that intended ?

Reply:
Yea, I'm reworking this a bit to record all failure cases (like that one).